runtimes/eden/src/pallets_system.rs

```diff
@@ -102,7 +102,7 @@ impl pallet_balances::Config for Runtime {
 }
 
 parameter_types! {
-	pub const TransactionByteFee: Balance = 1 * constants::MICRO_NODL;
+	pub const TransactionByteFee: Balance = constants::MICRO_NODL;
```

Comment:
I had suggested this change before. The thing is in this case clippy seems to be wrong. There is no extra overhead or unnecessary multiplication after this line is compiled, I have tested it. Then it goes back to the matter of style, which some prefer consistency (so this line looks similar to the previous lines), some would prefer conciseness. I'm not strongly in favour or against any of these two. I like to not see the clippy warning any way.